src/rust/bitbox02/src/event.rs

```diff
@@ -0,0 +1,7 @@
+pub use bitbox02_sys::event_slider_data_t;
```

Comment:
License header missing here and all other new files.

Reply:
fixed